config/values.yml

```diff
@@ -11,4 +11,4 @@ push_images: false
 image_cache: true
 image_repo: docker.io/k14stest/kapp-controller-test
 
-api_port: 10349
+api_port: 10350
```

Comment:
Due to how this is required by the start up code itself, should we add anything with ytt to make sure this field is specified? Maybe even think about using a simple schema here?